src/main/java/uk/gov/hmcts/reform/pip/account/management/model/Roles.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.pip.account.management.model;
+
+public enum Roles {
```

Comment:
Only minor however I'd add java docs on this enum (and user provenances)

Reply:
shall doozle!